doc/api/next_api_changes/behavior/18900-JMK.rst

```diff
@@ -0,0 +1,26 @@
+Axes used to make colorbar now wrapped
+======================================
+
+The axes used to place a colorbar is now wrapped by a new parent class
+when the colorbar is instnatiated ``ColorbarAxes``::
+
+    cb = fig.colorbar(im, cax=cax)
+
+So now ``cb.ax`` returns the parent ``ColorbarAxes`` and to get
+``cax`` back, you can do::
+
+    assert cb.ax.parent_axes == cax
```

Comment:
```suggestion
    cb.ax.parent_axes
```
I think using an assert here is a bit confusing, and just documenting the property fine.